hs_core/views/__init__.py

```diff
@@ -1849,15 +1856,20 @@ def _send_email_on_metadata_acceptance(request, shortkey):
     """
 
     resource = get_resource_by_shortkey(shortkey)
-    email_msg = f"""Dear Resource Owners,
-    <p>Your publication request for the following resource has been accepted:</p>
-    <p><a href="{ request.scheme }://{ request.get_host }/resource/{ resource.short_id }">{ request.scheme }://{ request.get_host }/resource/{ resource.short_id }</a></p>
-    
-    <p>Thank you</p>
+    email_msg = f'Dear Resource Owner, '\
```

Comment:
Use of """ for creating the message was better. I assume you had to replace it to use single quote to get the line breaks to work. Could we use  html line break tag `<br>` within """" string instead?

Reply:
thanks Pabitra, good call, I improved this...